documentation/src/main/asciidoc/topics/con_write_behind.adoc

```diff
@@ -1,15 +1,14 @@
 [id='write_behind']
 = Write-Behind
-Write-Behind is an cache writing mode where writes to memory are synchronous
-and writes to cache stores are asynchronous. When a client application
-updates a cache entry, {brandname} adds the update to a modification queue and
-then modifies the cache store in a different thread than the client thread.
-
-You can configure the number of threads that consume the modification queue and
-apply updates to the underlying cache store. The modification queue fills up if
-there are not enough threads to handle the updates or if the underlying cache
-store becomes unavailable. When this occurs, {brandname} uses Write-Through
-mode until the modification queue can accept new entries.
+Write-Behind is a cache writing mode where writes to memory are synchronous
+and writes to cache stores are asynchronous. When client applications
+update entries, {brandname} modifies cache stores in different threads to the
+client threads.
+
+When clients send write requests, {brandname} adds those operations to a
+modification queue until each previous write operation completes. If the 
+modification queue becomes full, subsequent write operations block the calling
+threads.
```

Comment:
@wburns hi, does this sound correct? tbh it's not entirely clear to me but I know we want more non-blocking behaviour so I have some doubts. thx.